packages/tocco-ui/src/EditableValue/StyledEditableValue.js

```diff
@@ -23,14 +23,15 @@ const StyledInputCss = css`
   background-color: transparent;
   border: 0;
   cursor: ${({immutable}) => immutable ? 'not-allowed' : 'default'};
+  pointer-events: ${({immutable}) => immutable ? 'none' : 'auto'};
```

Comment:
We still have at least one case in which the editable value should remain clickable: the value of a remotefield to open the record in a new tab.
But i tested it and the link still works. Why though?

Reply:
I thought the same thing. Theoretically the input field shouldn't be clickable since `pointer-events: none` will make the element completely unresponsive to a hover or click. I think it's because there is always another parent element wrapping around this css. I tested it on mobile and on disabled fields the touch is properly disabled.